src/openapi/peridio-admin-openapi.yaml

```diff
@@ -2,11 +2,17 @@
 openapi: 3.1.1
 info:
   title: Peridio Admin API
-  version: "1"
+  version: 2
   license:
     name: Peridio
     url: https://github.com/peridio/parasola/blob/main/LICENSE
   description: |
+    # Older APIs
```

Comment:
"Older APIs" feels odd.. maybe "Previous API Versions"?

Wish we could style this a bit more so it was more obvious to consumers, but we're probably limited by the OpenAPI renderer.